src/components/Breakfasts/ItemBreakfast.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react'
+
+
+const ItemBreakfast = ({ idItem, img, dish, price, breakfasts, order, setOrder }) => {
+
+
+    //Agregar los items seleccionados al state
+    const updateBreakfast = (idItem, price) => {
+        //console.log(idItem);
+        //Items seleccionados
+        const productSelect = breakfasts.filter(lun => lun.id === idItem)[0]
```

Comment:
```suggestion
        const productSelect = breakfasts.filter(lunch => lunch.id === idItem)[0]
```
más explicita la declaración